source/scripts/index.js

```diff
@@ -21,37 +11,12 @@ const firebaseConfig = {
 	storageBucket: 'dudeka-401e8.appspot.com',
 };
 
-const accentColor = '#ffe151';
-const accentDarkColor = '#f7cc00';
-const dudekaTheme = getMuiTheme({
-	datePicker: {
-		color: accentDarkColor,
-		textColor: '#333',
-		selectColor: accentColor,
-		selectTextColor: '#fff',
-	},
-	flatButton: {
-		textColor: '#3f95ff',
-		primaryTextColor: '#3f95ff',
-	},
-});
-
 injectTapEventPlugin();
 moment.locale('ru');
 firebase.initializeApp(firebaseConfig);
 
-function AppRoot() {
-	return (
-		<MuiThemeProvider muiTheme={dudekaTheme}>
-			<Provider store={store}>
-				<Routes history={syncHistoryWithStore(browserHistory, store)} />
-			</Provider>
-		</MuiThemeProvider>
-	);
-}
-
 function onDOMContentLoaded() {
-	ReactDOM.render(<AppRoot />, document.querySelector('#app'));
+	ReactDOM.render(<Root />, document.querySelector('#app'));
 }
 
 document.addEventListener('DOMContentLoaded', onDOMContentLoaded);
```

Comment:
Can we remove this?

Reply:
Yep, it's not necessary in this app. I removed it

Reply:
done in 016e32fb2b47866ce61be01f7963657400b0997d